### ISSUE TITLE
An unnecessary exception: "Cannot create consistent ordering methods"

### ISSUE THREAD

**Issue by danielpcampagna** [microsoft/pyright#568]
**Describe the bug**
The Pyright will throw an exception when I create classes inheriting from type function.
However, the Python interpreter does not.

**To Reproduce**
```python
X1 = type('X1', (object,), dict())
X2 = type('X2', (object,), dict())

class A(X1):
    ...
class B(X2, A):
    ...
```

**Expected behavior**
This following code works well:
```python
class X1:
    ...
class X2:
    ...
class A(X1):
    ...
class B(X2, A):
    ...
```

**Comment by erictraut** [comment]
Thanks for the bug report. Pyright doesn't currently know about the multi-parameter form of the built-in "type" call. I've added the necessary support, and this will be included in the next published version of Pyright.

**Comment by erictraut** [comment]
This is now fixed in version 1.1.28, which I just published.
